### DISCUSSION TITLE
Is there a way to install Script Kit via CLI?

### DISCUSSION THREAD

**Discussion by jacob-shuman** [johnlindquist/kit#896] [Q&A] [answered]
Is there currently anyway to install scriptkit via HomeBrew, MacPorts, etc? I'd love a way to automatically install Scriptkit if I need to reset my mac!

**Comment by johnlindquist** [comment, marked as answer]
Not currently, it's definitely been requested a few times, so I'll bump it up the priority list after this next release.

We're getting super close to the next big release if you wanna hop on our new Discord to try out the features or chat about anything else you'd like to see:

https://discord.gg/dEhUPpQU

**Reply by jacob-shuman** [reply]
That makes sense, thanks for the quick reply! I've been loving Script Kit and really appreciate all the work towards it.
